website/pages/en/index.js

```diff
@@ -74,115 +71,219 @@ HomeSplash.propTypes = {
+          title: language.name,
+          image: language.image,
+          imageAlign: "top",
+          content: language.variants.join("\n\n")
+        }))}
+        layout="fourColumn"
+      />
+    </Container>
+  </div>
+);
+
+const Editor = ({ content = "", image, name }) => (
+  <div style={{ display: "flex", width: "235px", padding: "20px" }}>
+    <img src={image} style={{ width: "100px", height: "100px" }} />
+    <div style={{ flexGrow: 1, paddingLeft: "8px" }}>
+      <h3 style={{ marginBottom: "-16px" }}>{name}</h3>
+      <MarkdownBlock>{content.replace(/\n/g, "\n\n")}</MarkdownBlock>
+    </div>
+  </div>
+);
+
+Editor.propTypes = {
+  content: React.PropTypes.string,
+  image: React.PropTypes.string.isRequired,
+  name: React.PropTypes.string.isRequired
+};
+
+const EditorSupportSection = () => (
+  <div className="editorSupportSection productShowcaseSection lightBackground paddingTop paddingBottom">
+    <Container>
+      <h2>Editor Support</h2>
+      <div
+        style={{
+          display: "flex",
+          flexFlow: "row wrap",
+          justifyContent: "space-around"
+        }}
+      >
+        {siteConfig.editors.map(editor => (
+          <Editor key={editor.name} {...editor} />
+        ))}
+      </div>
+    </Container>
+
+    <div style={{ float: "right" }}>
+      <span>Got more? </span>
+      <a
+        href={`${siteConfig.githubUrl}/edit/master/website/data/editors.yml`}
+        className="button"
+      >
+        Send a PR
+      </a>
+    </div>
+  </div>
+);
+
+const bash = (...args) => `~~~bash\n${String.raw(...args)}\n~~~`;
+
+const json = object => `~~~json\n${JSON.stringify(object, null, 2)}\n~~~`;
+
+class GetStartedSection extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      npmClient: "yarn"
+    };
+  }
+
+  render() {
+    return (
+      <div className="getStartedSection productShowcaseSection paddingTop paddingBottom">
+        <Container>
+          <h2>Get Started</h2>
+          <div>
+            <ol>
+              <li>
+                Add prettier to your project:
+                <MarkdownBlock>
+                  {bash`yarn add prettier --dev --exact`}
+                </MarkdownBlock>
+              </li>
+              <li>
+                Verify by running against a file:
+                <MarkdownBlock>
+                  {bash`yarn prettier --write src/index.js`}
+                </MarkdownBlock>
+              </li>
+              <li>
+                Run prettier when commiting files:
+                <MarkdownBlock>
+                  {bash`yarn add pretty-quick husky --dev`}
+                </MarkdownBlock>
+                Then edit <code>package.json</code>:
+                <MarkdownBlock>
+                  {json({
+                    scripts: {
+                      precommit: "pretty-quick --staged"
+                    }
+                  })}
+                </MarkdownBlock>
+              </li>
```

Comment:
I'd be really curious to know how many people use prettier as a pre-commit hook and how many instead use ESLint to run prettier. Personally I'm using ESLint because we were already using it and it was already integrated with our code review and CI system. Given the questions we get it seems I'm far from alone.

Reply:
This one I'll leave you to all you core devs to figure out, I opted for lint-staged as it's the one I've used everywhere and works in all environments

However, it's totally possible to play with this section to also let it support many options too

e.g. (60s mockup)

<img width="844" alt="screen shot 2018-01-11 at 4 04 16 pm" src="https://user-images.githubusercontent.com/49038/34847381-1d766790-f6e9-11e7-917f-130d0a0a6548.png">